.github/workflows/pull-request.yaml

```
@@ -0,0 +1,48 @@
+name: Pull Request
+
+on:
+  pull_request:
+    tyeps: [ opened, synchronize, reopened ]
```

actionlint (1.7.7)

unexpected key "tyeps" for "pull_request" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)